clang/include/clang/Basic/Builtins.td

```diff
@@ -4782,6 +4782,12 @@ def HLSLStep: LangBuiltin<"HLSL_LANG"> {
   let Prototype = "void(...)";
 }
 
+def HLSLFmod : LangBuiltin<"HLSL_LANG"> {
+  let Spellings = ["__builtin_hlsl_elementwise_fmod"];
```

Comment:
rename to `__builtin_elementwise_fmod` the add appropriate changes to `clang/docs/LanguageExtensions.rst` and  `clang/docs/ReleaseNotes.rst`

Reply:
Done.